src/typeDefinitions/default.types.d.ts

```diff
@@ -2,6 +2,14 @@ export interface env {
   [key: string]: string;
 }
 
+export interface environment {
+  [key: string]: variables;
+}
```

Comment:
cant we use the env directly as it follows same type definitions

Reply:
Nope because we have object inside object and then we have string while for the `env` it one object having properties and values as strings